includes/html/graphs/device/storage.inc.php

```diff
@@ -27,7 +27,7 @@
         $iter   = '0';
     }
 
-    $descr        = rrdtool_escape($storage['storage_descr'], 12);
+    $descr        = rrdtool_escape($storage['storage_descr'], 16);
```

Comment:
Other graphs also seems to have 12 chars, like: https://github.com/librenms/librenms/blob/master/includes/html/graphs/storage/usage.inc.php#L19  
If we change one, we should change all?

Reply:
@Jellyfrog 
i agree.
is also changed now.